frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts

```diff
@@ -40,7 +40,8 @@ import {LoadingIndicatorService} from "core-app/modules/common/loading-indicator
 import {HalResourceNotificationService} from "core-app/modules/hal/services/hal-resource-notification.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/2ae7cbab6a846f63f4c17f118ded9c3201310783/frontend/src/app/modules/boards/new-board-modal/new-board-modal.component.ts#L41"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>